app/test_main.py

```diff
@@ -1,3 +1,19 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17])
+    ]
+)
+def test_get_human_age(cat_age: int, dog_age: int, result: list[int]) -> None:
+    assert get_human_age(cat_age, dog_age) == result
```

Comment:
Add also negative tests cases : ) I mean where you expecting some error ot something